### PR TITLE
add: liquid gas token

### PR DESCRIPTION
**Pull request by sambacha**
ya forgot one: [https://github.com/matnad/liquid-gas-token](https://github.com/matnad/liquid-gas-token)

### CODE REVIEW

**Comment by wjmelements** [comment]
LGT doesn't implement the same interface; the args are switched.

**Review by wjmelements — CHANGES_REQUESTED** [review]
Need to define ILiquidGasToken and add a separate method for it.

File: contracts/Farts.sol [hunk 1]
```diff
@@ -12,6 +12,7 @@ contract Farts /*is IERC20*/ {
     IGasToken public constant GST1 = IGasToken(0x88d60255F917e3eb94eaE199d827DAd837fac4cB);
     IGasToken public constant GST2 = IGasToken(0x0000000000b3F879cb30FE243b4Dfee438691c04);
     IGasToken public constant CHI = IGasToken(0x0000000000004946c0e9F43F4Dee607b0eF1fA1c);
+    IGasToken public constant LGT = IGasToken(0x000000000000C1CB11D5c062901F32D06248CE48); /** Liquid Gas Token */
```
Comment (wjmelements) [inline contracts/Farts.sol:15]: If you're adding this comment, add it for the others too

**Comment by sambacha** [comment]
> Need to define ILiquidGasToken and add a separate method for it.

sounds like ya got some work ahead of ya

**Comment by sambacha** [comment]
> LGT doesn't implement the same interface; the args are switched.

i only used it once i swear

**Comment by wjmelements** [comment]
I'm gonna reject LGT because of low supply and market cap. The different `freeFrom` interface means it's not worth supporting.

**Pull request closed without merging**